content/save-load-arrays.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>It might be worth citing <a href="https://numpy.org/devdocs/user/basics.io.genfromtxt.html" target="_blank">Importing with genfromtxt</a> and <a href="https://numpy.org/devdocs/user/how-to-io.html" target="_blank">Ben's how to</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Yes, I completely forgot to give next steps. How about:</p><blockquote>Next steps, you can import data with missing values from <a href="https://numpy.org/doc/stable/user/basics.io.genfromtxt.html" target="_blank">Importing with genfromtext</a> or learn more about general NumPy IO with <a href="https://numpy.org/devdocs/user/how-to-io.html" target="_blank">Reading and Writing Files</a>. </blockquote><p>Those links are for devdocs,, @mattip suggested only using the stable docs links. </p>

Reply:
<p>As part of the next steps, I also wanted to include a link to the mooreslaw.ipynb, but we can changes links and refs in a future PR. </p>

Reply:
<p>Oops yeah, use the stable links!</p>